pages/web-literacy/WebLiteracyPage.jsx

```diff
@@ -1,44 +1,314 @@
+            <label className="checkbox-label" htmlFor={cat + "-checkbox"}>
+              <span className="custom-checkbox">
+                <i className="fa fa-check"></i>
+              </span>
+              { categories[cat] }
+            </label>
+          </span>
+        </li>
+      );
+    }.bind(this));
+  },
+  renderTopics: function() {
+    if (this.state.topic) {
+      return null;
+    }
+    return Object.keys(weblitdata).map(function(topic) {
+      return (
+        <div key={topic}>
           <Illustration
-          width={210} height={210}
-          src1x="/img/pages/web-literacy/svg/icon-weblit.svg"
-          alt="web literacy illustration">
-            <h2>
-              Web Literacy is the skills and competencies needed for reading, writing, and participating on the Web.
-            </h2>
+            width={150} height={150}
+            src1x={topicContent[topic].imgSrc1x}
+            src2x={topicContent[topic].imgSrc2x}
+            alt={"weblit-map-icon-" + topic}>
+            <h2>{topic}</h2>
+            <p>{topicContent[topic].content}</p>
+            <span><b>Web Literacy Skills:</b> {makeLinksFromWebLitSkills(Object.keys(weblitdata[topic]))}</span>
           </Illustration>
-        </section>
-
-        <section>{webmaps}</section>
+        </div>
+      );
+    });
+  },
+  renderTopic: function() {
+    if (!this.state.topic || this.state.webLitSkill) {
+      return null;
+    }
+    return (
+      <div className="topic-container">
+        <h3>{this.state.topic}</h3>
+        <p><img src={topicContent[this.state.topic].imgSrc1x} width={75} height={75}/>{topicContent[this.state.topic].content}</p>
+      </div>
+    );
+  },
+  renderWebLitSkill: function() {
+    if (!this.state.topic || !this.state.webLitSkill) {
+      return null;
+    }
+    return (
+      <div className="web-lit-skill">
+        <h3><span className="lighten">{this.state.topic} &rsaquo;</span> {this.state.webLitSkill}</h3>
+        <p>{webLitSkillsContent[this.state.webLitSkill].quote}</p>
+      </div>
+    );
+  },
+  renderCompetencies: function() {
+    if (!this.state.topic || !this.state.webLitSkill) {
+      return null;
+    }
+    return (
+      <div className="web-lit-competency">
+        <h3>{this.state.webLitSkill} Competencies</h3>
+        {
+          webLitSkillsContent[this.state.webLitSkill].content.map(function(value, index) {
+            return (
+              <p className="web-lit-skill-content" key={index}>
+                {value}
+              </p>
+            );
+          })
+        }
+      </div>
+    );
+  },
+  onMapToggle: function(labels) {
+    var verb =  labels[1] || "";
+    var webLitSkill = labels[2] || "";
+    var url = "/web-literacy/";
+    if (verb) {
+      url += verb + "/";
+      if (webLitSkill) {
+        url += webLitSkill + "/";
+      }
+    }
+    this.history.pushState(null, url);
```

Comment:
I thought we had access to `this.context.history`, which would allow use to not need a history mixin and would let us say
```
this.context.history.push({
  pathname: '/the/path',
  search: '?a=query',
  state: { the: 'state' }
}); 
```
(the history that React Router relies on should get autobound if you specify `contextTypes: { history: React.propTypes.object }` as part of the component definition)